src/Task.ts

```diff
@@ -179,6 +183,7 @@ export class Task {
         tags: string[] | [];
         originalMarkdown: string;
         scheduledDateIsInferred: boolean;
+        createdDate: moment.Moment | null;
```

Comment:
Where you have added createdDate fields and parameters throughout, please can you keep the order consistent. It seems like in some places it is added after existing related fields, and in other places it is before.

Lower down, I see it is just after `doneDate` which is fine, and consistency with that would be very helpful.

Reply:
It will still need to be the last argument in the Tasks constructor, though, I think...

Reply:
Every place that I've noticed, move createdDate above startDate.